jarbas_hive_mind/database/json_db.py

```diff
@@ -104,7 +104,7 @@ def get_clients_by_name(self, name):
         return self.search_by_value("name", name)
 
     def add_client(self, name=None, mail=None, key="",
-                   admin=None, blacklist=None, crypto_key="RESISTENCEisFUTILE"):
+                   admin=None, blacklist=None, crypto_key=None):
```

Comment:
unrelated bugfix, did not allow NOT having a crypto key, sounds like crypto was accidentally enforced